photos/db.go

```diff
@@ -258,8 +258,11 @@ func listPhotos(tx *sql.Tx, userID int, photoIDs *[][]byte) error {
 	if err != nil {
 		return err
 	}
-	const selectSQL = `
-SELECT id, caption, commentCount, latitude, longitude, timestamp FROM photos WHERE userID = $1 ORDER BY timestamp DESC LIMIT 100`
+	const selectSQL = `SELECT commentID, userID, message, timestamp FROM comments ` +
```

Comment:
BTW Specifically Peter's suggestion is to change commentsByPhotoID to be:

`CREATE UNIQUE INDEX IF NOT EXISTS commentsByPhotoID ON comments (photoID, timestamp) STORING (commentID, userID, message);`

Verified locally this would do the right thing:
```
+-------+---------+------------------------------------+
| Level |  Type   |            Description             |
+-------+---------+------------------------------------+
|     0 | revscan | comments@commentsByPhotoID /10-/11 |
+-------+---------+------------------------------------+
```

But this means we will be storing each message twice. So I'm no longer sure which one is better :)